A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -1494,17 +1540,16 @@ var A320_Neo_UpperECAM;
         getActiveFailures() {
             let output = {};
             this.hasActiveFailures = false;
-            this.hasWarnings = false;
+                        this.hasWarnings = false;
             this.hasCautions = false;
             for (var i = 0; i < this.messages.failures.length; i++) {
                 const messages = this.messages.failures[i].messages;
                 for (var n = 0; n < messages.length; n++) {
                     const message = messages[n];
                     if (message.id == null) message.id = `${i} ${n}`;
                     if (message.isActive() && !this.clearedMessages.includes(message.id)) {
+                    if (message.isActive()) {
                         this.hasActiveFailures = true;
-                        if (message.level == 3) this.hasWarnings = true;
-                        if (message.level == 2) this.hasCautions = true;
                         if (output[i] == null) {
```

Comment:
did you mean to touch this?

Reply:
was left out in the last fix, added it with the formatting fix.